### PR TITLE
[5.2][CodeCompletion] Stop printing underscored keyword in override completion

### PR DESCRIPTION
**Pull request by rintaro**
Cherry-pick of #29127 into `swift-5.2-branch` originally reviewed by @nathawes

rdar://problem/56963545

### CODE REVIEW

**Comment by rintaro** [comment]
The added test case depends on https://github.com/apple/swift/pull/29104. Holding off until it's merged

**Comment by DougGregor** [comment]
@swift-ci please test

**Review by DougGregor — APPROVED** [review]

**Pull request merged**
